### DISCUSSION TITLE
Bark Coding's new name

### DISCUSSION THREAD

**Discussion by mariocraft987** [cytrinc/bark-coding#45] [Polls]
"Bark Coding" seems too simple, so i'm thinking up of another name!

**Comment by raynepaws** [comment]
barkly (twist on "blockly")

**Comment by mariocraft987** [comment]
Barkly coding

**Reply by raynepaws** [reply]
yes

**Reply by rubiidev18alt** [reply]
you know what's better, blarkly.

**Reply by raynepaws** [reply]
> you know what's better, blarkly.

isn't that straying a little too far from "bark"? :/

**Reply by mariocraft987** [reply]
Blarkly sounds like a clay company
